go/src/koding/kites/config/cache.go

```diff
@@ -105,6 +105,10 @@ func KodingCacheHome() string {
 		cache = filepath.Join(CurrentUser.HomeDir, ".cache", "koding")
 	}
 
+	// Best-effort attempt, ignore errors.
+	_ = os.MkdirAll(cache, 0755)
+	_ = util.Chown(cache, CurrentUser.User)
```

Comment:
I'm not convinced this is the right place.

I'd rather add it [here](https://github.com/koding/koding/blob/master/go/src/koding/klientctl/install.go#L286) and [here](https://github.com/koding/koding/blob/master/go/src/koding/klientctl/update.go#L211-L212).